src/settings.plugin.js

```diff
@@ -257,6 +247,118 @@ wikiaJWPlayerSettingsPlugin.prototype.updateCurrentQuality = function (data) {
 	}
 };
 
+wikiaJWPlayerSettingsPlugin.prototype.addCaptionListener = function () {
+	var clickHandler = this.captionsClickHandler.bind(this);
+
+	this.player.on('captionsList', function (event) {
+		// tracks always include "off" item
+		if (event.tracks.length > 1) {
+			this.currentlySelectedCaptions = this.getSuitableCaptionsIndex(
+				this.captionLangMap[this.getUserLang()],
+				event.tracks
+			);
+
+			this.getLabelElement(this.captionsToggle)
+				.addEventListener('click', clickHandler);
+
+			this.wikiaSettingsElement.classList.remove('are-captions-empty');
+			this.player.setCurrentCaptions(this.currentlySelectedCaptions);
+			this.show();
+		} else {
+			this.getLabelElement(this.captionsToggle)
+				.removeEventListener('click', clickHandler);
+
+			this.wikiaSettingsElement.classList.add('are-captions-empty');
+		}
+	}.bind(this));
+};
+
+wikiaJWPlayerSettingsPlugin.prototype.createCaptionsButton = function () {
+	this.captionsToggle = createToggle({
+			id: this.player.getContainer().id + '-videoCaptionsToggle',
+			label: 'Captions',
+			checked: this.config.showCaptionsToggle
+		});
+
+	this.captionsToggle.classList.add('wikia-jw-settings__captions-button');
+
+	return this.captionsToggle;
+};
+
+wikiaJWPlayerSettingsPlugin.prototype.captionsClickHandler = function () {
+	if (this.areCaptionsOff(this.player.getCurrentCaptions())) {
+		this.player.setCurrentCaptions(this.currentlySelectedCaptions)
+	} else {
+		// "off" caption track is always the first one
+		this.player.setCurrentCaptions(0);
+	}
+};
+
+wikiaJWPlayerSettingsPlugin.prototype.areCaptionsOff = function(captionIndex) {
+	// "off" always has first position in tracks array
+	return captionIndex === 0;
+};
+
+wikiaJWPlayerSettingsPlugin.prototype.getUserLang = function() {
+	return window.navigator.language.slice(0, 2);
```

Comment:
Does IE11 support it? 
`https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language`

Reply:
from what i read - it should, but I've added fallback anyway